### PR TITLE
add how-to for OCI image layout

### LINKED ISSUE #166: Create tutorials for experimental feature sign/verify images in oci image layout directory
`sign/verify images in OCI image layout directory` is the new experimental feature for RC-4. A tutorial guide is needed for users to understand usage of this feature. This guide will be placed under https://github.com/notaryproject/notaryproject.dev/tree/main/content/en/docs/tutorials

### PR DESCRIPTION
**Pull request by zr-msft**
fixes #166

### CODE REVIEW

**Review by yizha1 — COMMENTED** [review]
@zr-msft Thanks Zach. A few comments were added.

File: content/en/docs/how-to/oci-image-layout.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+{{% /alert %}}
+
+Use `notation list` and `notation verify` to list and verify the image signatures. For example:
+
+```console
+notation list localhost:5001/hello-world:v1
+notation verify localhost:5001/hello-world:v1
+```
+
+## Download an image as an OCI image layout
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/docs/how-to/oci-image-layout.md:98]: @yizha1 

This isn't an end-to-end tutorial, this is a general how-to covering the the different features/aspects of OCI image layout in the context of notation. 

Downloading an image as OCI image layout from a registry is called out in the hackmd doc you shared in #166 :
https://hackmd.io/8MLW8iQCTI-WNwvYb9Vykw#Copy-remote-image-to-the-disk-in-the-form-of-OCI-image-layout-Option-2

Do you think users need to know how to do this? For example, will a user need to download an image from a registry as an OCI layout for signing or debugging purposes? If not, i can remove this section

File: content/en/docs/how-to/oci-image-layout.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+Use `notation list --oci-layout` to list signatures associated with an OCI image layout.
+
+```shell
+notation list --oci-layout ./hello-world:v1
+```
+
+Use `notation verify --oci-layout` to verify the image against signatures.
+
+```shell
+notation verify --oci-layout ./hello-world:v1 --scope "local/hello-world"
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/docs/how-to/oci-image-layout.md:68]: Thank you @yizha1 

When reviewing the hackmd, there wasn't much context around the example trust policy so I omitted it:
https://hackmd.io/8MLW8iQCTI-WNwvYb9Vykw#Sign-and-verify-OCI-images-on-local-disk

I'm going to refactor this portion and make a *Verify OCI image* section separate from signing.

[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/docs/how-to/oci-image-layout.md:97]: @yizha1 updated the PR to incorporate the refactor

File: content/en/docs/how-to/oci-image-layout.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (yizha1) [reply content/en/docs/how-to/oci-image-layout.md:127]: I suggest removing this part for now. If the images are already in the registries, users can sign it directly, it is rare case that users download it first, then sign it. We can add it back if there is a real use case. That part in hackmd is to show different options of getting the OCI image layout, which seems not a real use case now.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**